### DISCUSSION TITLE
Data Talk

### DISCUSSION THREAD

**Discussion by PBissonDFG** [Data-For-Good-Ottawa/OFB_2023#3] [General]
In the repo you will find two password protected Zip files.
These zip files contain a questionar (.docx) and its coresponding responses (.csv 65001: Unicode (UTF-8)). 

The two surveys are called "Hunger Count" and "Neighbour Survey".

"Neighbour Survey" has 4086 records and 136 fields.
"Hunger Count" has 85 records and 49 fields

The Neighbour Survey has an id field, 20 text fields,  24 categorical fields and 91 Boolean fields, 
while "Hunger Count" has an id field, 11 text fields, 32 categorical fields, 5 Boolean fields

**Comment by murphy99** [comment]
Hi PBissonDFG,

There seems to be missing records between records 3969-3985 in the neighbourhood_survey_clean-2024-06-14.csv file. Also, duplicate records, example id= NS 3980.

<img width="1551" alt="Screenshot 2024-06-21 at 12 22 01 PM" src="https://github.com/Data-For-Good-Ottawa/OFB_2023/assets/1928632/bf9b61a4-182c-4009-b238-cf1d5d05ba65">

Regards, Eileen
